graphene/src/boolean_ops.rs

```diff
@@ -55,7 +48,17 @@ struct Vertex {
 
 impl Debug for Vertex {
 	fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
-		f.write_str(format!("\n    Intersect@ {:?}", self.intersect.point).as_str())?;
+		f.write_str(
+			format!(
+				"\n    Intersect@ {:?} a_seg: {:?}, b_seg: {:?} t_a: {:?} t_b: {:?}",
```

Comment:
- Why the 4 spaces at the start of the string? If they do in fact belong, adding a comment explaining this question would be helpful.
- Is that at sign supposed to be a colon in `Intersect@`?
- the `_seg` and `t_` suffixes and prefixes are pretty cryptic to me. Could you pick full words that describe exactly what these refer to?
Thanks!